app/poros/error.rb

```diff
@@ -61,5 +61,14 @@ def invalid_parameter(message, metadata = {})
         metadata: metadata
       )
     end
+
+    def book_already_rented
```

Comment:
Pudieras mejorar el mensaje declarando que `book_id` ya está rentado

```suggestion
    def book_already_rented(book_id)
```